devkit/src/Runner.ts

```diff
@@ -310,6 +310,7 @@ export class Runner {
         return {
           status: TestStepResultStatus.AMBIGUOUS,
           duration: TimeConversion.millisecondsToDuration(0),
+          message: error.message,
```

Comment:
Does the error message have to be included? It can be derived from the multiple pickle matches.

Note: Cucumber JVM does currently include the message.